templates/front_page.html

```diff
@@ -27,7 +27,7 @@
                     Projects
                 </div>
             </a>
-            <img src="{{ url_for('static', filename='projects_card_img.jpg')}}"
+           <a href="https://goo.gl/images/shx1pQ"></a>
```

Comment:
Image links should be put in `<img>` tag, not in the anchor `<a>` tag
Anchor tag creates a link with a text
It doesn't add image to the document